spec/rmagick/image/import_pixels_spec.rb

```diff
@@ -38,4 +45,71 @@
       img.import_pixels(0, 0, 20, 20, 'RGB', pixels, Magick::UndefinedPixel)
     end.to raise_error(ArgumentError, /UndefinedPixel/)
   end
+
+  it 'works' do
```

Comment:
Should this have another name?

Reply:
@dlemstra it should. Nothing immediately came to mind, though, so I punted until it gets revisited in a followup to break it down more. If you've got a suggestion I'd be happy to update it.

Reply:
@dlemstra I went ahead and updated it with a different description.